### ISSUE TITLE
[v0.36.0] `yarn rw storybook` is broken in main branch

### ISSUE THREAD

**Issue by peterp** [redwoodjs/redwood#3115]
Steps to reproduce:

```terminal
./tasks/run-e2e
cd /path/to/redwood/project
yarn rw sb

[... errors ...]
TypeError: The 'compilation' argument must be an instance of Compilation
[... errors ...]
```

**Comment by thedavidprice** [comment]
@peterp Thanks for catching this. Any chance you know when/how this changed — and/or did you remove storybook test from Cypress? I'm curious how it made it through CI E2E... e.g.  🤔

Will try to diagnose tonight/tomorrow. Need to resolve before v0.36.

**Comment by peterp** [comment]
@thedavidprice I'm not sure when it broke, the e2e tests run `yarn rw sb --build` which still works! (lol?)

**Comment by thedavidprice** [comment]
Fixed in #3132
